js/controller.js

```diff
@@ -753,6 +753,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.renderSkin();
       }
       else if (videoId == OO.VIDEO.ADS){
+        //If we pause during an ad (such as for clickthroughs or when autoplay fails)
+        //we'll show the control bar so that the user has an indication that the video
+        //must be unpaused to resume
+        this.state.config.adScreen.showControlBar = true;
```

Comment:
Do we need to worry about the autohiding, or will it show again if click the screen after it hides?

Reply:
It autohides by itself. It will appear from then on when the user hovers over the video.